### ISSUE TITLE
AmbiguousOperatorPrecedence wants double parentheses

### ISSUE THREAD

**Issue by francisdb** [rubocop/rubocop#10167]
AmbiguousOperatorPrecedence wants double parentheses here

```
spec/helpers.rb:38:35: W: [Correctable] Lint/AmbiguousOperatorPrecedence: Wrap expressions with varying precedence with parentheses to avoid ambiguity.
$wait_factor = $max_total_wait / (2**($max_attempts - 1) - 1)
                                  ^^^^^^^^^^^^^^^^^^^^^^
```

it wants looks like this

```ruby
$wait_factor = $max_total_wait / ((2**($max_attempts - 1)) - 1)
```

--------

## Expected behavior

Rubocop does not complain

## Actual behavior

See above

## Steps to reproduce the problem

Put the expression above in your code

## RuboCop version

```
> rubocop -V
Ignoring jaro_winkler-1.5.4 because its extensions are not built. Try: gem pristine jaro_winkler --version 1.5.4
1.21.0 (using Parser 3.0.2.0, rubocop-ast 1.11.0, running on ruby 2.6.3 universal.x86_64-darwin20)
  - rubocop-rspec 2.5.0
```

also the case for `1.22.0`
